misc/USB-Interface-bcu1/USB-IF_Usb/src/hid_knx.cpp

```diff
@@ -189,28 +189,28 @@ ErrorCode_t KnxHidIf::ReadReport(int &buffno)
  * EMI-Type, Versionsnummern und Co.
  * Parameter
  * ProtId: Protocol Identifier
- *          - C_TPH_PId_KnxTunnel
- *          - C_TPH_PId_BAS
+ *          - TPH_ProtocolID::knxTunnel
+ *          - TPH_ProtocolID::busAccessServer
  * PayloadLen: Payload Length, Laenge des Transfer Protocol Body
  * EmiServiceId:
- *          - Service Identifier bei C_TPH_PId_BAS
- *          - EMI ID bei C_TPH_PId_KnxTunnel
+ *          - Service Identifier bei TPH_ProtocolID::busAccessServer
+ *          - EMI ID bei TPH_ProtocolID::knxTunnel
  * Der zurueckgegebene Pointer zeigt auf das erste Byte des
  * KNX Transfer Protocol Body.
  */
 uint8_t* KnxHidIf::BuildUsbPacket(uint8_t *ptr, uint8_t ProtId, uint8_t PayloadLen, uint8_t EmiServiceId)
 {
   *ptr++ = 0x01; // Report Identifier
   *ptr++ = 0x13; // Packet Info - nur Single Packets unterstuetzt im Moment
-  *ptr++ = C_TPH_HeadLen+PayloadLen;
+  *ptr++ = TPH_ProtocolLength_V0+PayloadLen;
   *ptr++ = 0;
-  *ptr++ = C_TPH_HeadLen;
+  *ptr++ = TPH_ProtocolLength_V0;
   *ptr++ = 0;
   *ptr++ = PayloadLen;
   *ptr++ = ProtId;
   *ptr++ = EmiServiceId;
-  *ptr++ = C_TPH_ManuCode1;
-  *ptr++ = C_TPH_ManuCode2;
+  *ptr++ = TPH_ManufacturerCode_V0;
+  *ptr++ = TPH_ManufacturerCode_V0;
```

Comment:
`TPH_ManufacturerCode_V0` is a `uint16_t`, so I guess this misses usages of `HIGH_BYTE` and `lowByte`.

Reply:
Thx. Fixed it the simple way in 7b531da because the `USB_IF_Usb` project doesn´t use the sblib.